Arrowgene.Ddon.Shared/Network/PacketId.cs

```diff
@@ -928,18 +928,18 @@ public override int GetHashCode()
         public static readonly PacketId S2C_FRIEND_APPLY_FRIEND_RES = new PacketId(16, 1, 2, "S2C_FRIEND_APPLY_FRIEND_RES", ServerType.Game, PacketSource.Server); // フレンド申請に
-        public static readonly PacketId S2C_FRIEND_16_1_16_NTC = new PacketId(16, 1, 16, "S2C_FRIEND_16_1_16_NTC", ServerType.Game, PacketSource.Server);
+        public static readonly PacketId S2C_FRIEND_APPLY_FRIEND_NTC = new PacketId(16, 1, 16, "S2C_FRIEND_16_1_16_NTC", ServerType.Game, PacketSource.Server);
         public static readonly PacketId C2S_FRIEND_APPROVE_FRIEND_REQ = new PacketId(16, 2, 1, "C2S_FRIEND_APPROVE_FRIEND_REQ", ServerType.Game, PacketSource.Client);
         public static readonly PacketId S2C_FRIEND_APPROVE_FRIEND_RES = new PacketId(16, 2, 2, "S2C_FRIEND_APPROVE_FRIEND_RES", ServerType.Game, PacketSource.Server); // フレンド承認に
-        public static readonly PacketId S2C_FRIEND_16_2_16_NTC = new PacketId(16, 2, 16, "S2C_FRIEND_16_2_16_NTC", ServerType.Game, PacketSource.Server);
+        public static readonly PacketId S2C_FRIEND_APPROVE_FRIEND_NTC = new PacketId(16, 2, 16, "S2C_FRIEND_16_2_16_NTC", ServerType.Game, PacketSource.Server);
         public static readonly PacketId C2S_FRIEND_REMOVE_FRIEND_REQ = new PacketId(16, 3, 1, "C2S_FRIEND_REMOVE_FRIEND_REQ", ServerType.Game, PacketSource.Client);
         public static readonly PacketId S2C_FRIEND_REMOVE_FRIEND_RES = new PacketId(16, 3, 2, "S2C_FRIEND_REMOVE_FRIEND_RES", ServerType.Game, PacketSource.Server); // フレンド解除に
-        public static readonly PacketId S2C_FRIEND_16_3_16_NTC = new PacketId(16, 3, 16, "S2C_FRIEND_16_3_16_NTC", ServerType.Game, PacketSource.Server);
+        public static readonly PacketId S2C_FRIEND_REMOVE_FRIEND_NTC = new PacketId(16, 3, 16, "S2C_FRIEND_16_3_16_NTC", ServerType.Game, PacketSource.Server);
         public static readonly PacketId C2S_FRIEND_REGISTER_FAVORITE_FRIEND_REQ = new PacketId(16, 4, 1, "C2S_FRIEND_REGISTER_FAVORITE_FRIEND_REQ", ServerType.Game, PacketSource.Client);
         public static readonly PacketId S2C_FRIEND_REGISTER_FAVORITE_FRIEND_RES = new PacketId(16, 4, 2, "S2C_FRIEND_REGISTER_FAVORITE_FRIEND_RES", ServerType.Game, PacketSource.Server); // お気に入りフレンド設定に
         public static readonly PacketId C2S_FRIEND_CANCEL_FRIEND_APPLICATION_REQ = new PacketId(16, 5, 1, "C2S_FRIEND_CANCEL_FRIEND_APPLICATION_REQ", ServerType.Game, PacketSource.Client);
         public static readonly PacketId S2C_FRIEND_CANCEL_FRIEND_APPLICATION_RES = new PacketId(16, 5, 2, "S2C_FRIEND_CANCEL_FRIEND_APPLICATION_RES", ServerType.Game, PacketSource.Server); // フレンド申請キャンセルに
-        public static readonly PacketId S2C_FRIEND_16_5_16_NTC = new PacketId(16, 5, 16, "S2C_FRIEND_16_5_16_NTC", ServerType.Game, PacketSource.Server);
+        public static readonly PacketId S2C_FRIEND_CANCEL_FRIEND_APPLICATION_NTC = new PacketId(16, 5, 16, "S2C_FRIEND_16_5_16_NTC", ServerType.Game, PacketSource.Server);
```

Comment:
Add the original names of the identified packets as the last argument of the constructor

Reply:
ok should be good now